omero/developers/Web/LinkingFromWebclient.txt

```diff
@@ -0,0 +1,189 @@
+
+Linking from Webclient
+======================
+
+If you want users to be able to access your app or other resources from the webclient
+there are a number of ways you can add links to the webclient UI.
+
+OMERO.web top links
+-------------------
+
+You can configure :property:`omero.web.ui.top_links` to add links to the list
+of links at the top of the webclient main pages.
+
+-  **Name your URL in urls.py** (optional). Preferably we use URL names to
+   refer to URLs.
+   For example, the homepage of your app might be named like this in yourapp/urls.py.
+
+   ::
+
+       url( r'^$', views.index, name='figure_index' ),
+
+
+   You can then refer to the link defined above using this ``name``, or you can simply use a full URL for external links.
+
+
+-  **Update configuration** Use the OMERO command line interface to append the link to the ``top_links`` list.
+
+   Links use the format ``["Label", "URL_name"]`` or you can follow this example:
+
+   ::
+
+       $ bin/omero config append omero.web.ui.top_links '["Figure", "figure_index"]'
+
+   From OMERO 5.1, you can add additional attributes to links using the format ``['Link Text', 'link', attrs]``.
+   This can be used to add tool-tips and to open the link in a new "target" tab. For example:
+
+   ::
+
+       $ bin/omero config append omero.web.ui.top_links '["Homepage", "http://myhome.com", {"title": "Homepage", "target": "_blank"}]'
+
+
+Custom image viewer
+-------------------
+
+If you have created your own image viewer and would like to have it replace the existing image viewer in the
+webclient, this can be configured using :property:`omero.web.viewer.view`.
+
+You will need your :file:`views.py` method to take an Image ID with a parameter named ``iid``. For example, see
+``channel_overlay_viewer`` from the `webtest <https://github.com/openmicroscopy/webtest/>`_ app:
```

Comment:
Actually, as per the dev_5_2 branch, the link should be `OMERO.webtest` with the updated omero-webtest URL